tools/create-genesis/src/genesis.rs

```diff
@@ -1,17 +1,19 @@
+use std::collections::BTreeMap;
+use std::fs::File;
+use std::io::Write;
+use std::path::Path;
+use std::str::FromStr;
+
 use crate::common::{clean_0x, string_2_bytes};
```

Comment:
we have a `clean_0x` function in cita_types, why not use it?

Reply:
Done